CMakeLists.txt

```diff
@@ -78,10 +78,24 @@ set_optimization()
 ## Dynawo and sub-dependencies
 check_env_var(DYNAWO_HOME) # Correspond to deploy installation
 list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}")
+list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/cmake")
 list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/share")
-find_package(Dynawo 1.3.0 REQUIRED)
-#libXML
 list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/share/cmake")
+list(APPEND CMAKE_MODULE_PATH "${DYNAWO_HOME}/cmake")
+list(APPEND CMAKE_MODULE_PATH "${DYNAWO_HOME}/share/cmake")
+
+find_package(Dynawo 1.3.0 REQUIRED)
```

Comment:
dynawo should be place after its dependencies

Reply:
OK